task13/contacts.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lora&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="grid.css">
+    <script src="https://kit.fontawesome.com/9de98033f0.js" crossorigin="anonymous"></script>
+    <title>grid</title>
+</head>
+<body>
+<header>
+    <div class="logo"></div>
+    <div class="icons">
```

Comment:
For the same classes from both pages you can create new .css file with common styles.

Reply:
Yes) ofcorse)